easybuild/easyconfigs/d/DeepLabCut/DeepLabCut-2.3.6-foss-2022a-CUDA-11.7.0.eb

```diff
@@ -0,0 +1,74 @@
+# Loosely based on PR #7680 
+# J. Sassmannshausen (Imperial College London/UK)
+# upgrade to version 2.3.6: J Hein, (LUNARC, Lund University, Sweden)
+
+easyblock = 'PythonBundle'
+
+name = 'DeepLabCut'
+version = '2.3.6'
+versionsuffix = '-CUDA-%(cudaver)s'
+local_version = '%(versionsuffix)s-contrib'
+
+homepage = 'http://www.mousemotorlab.org/deeplabcut'
+description = "Markerless tracking of user-defined features with deep learning"
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'pic': True}
+
+dependencies = [
+    ('CUDA', '11.7.0', '', SYSTEM),
+    ('Python', '3.10.4'),
+    ('TensorFlow', '2.9.1',  versionsuffix),
```

Comment:
I'm a bit confused as to why the easyconfigs test suite isn't tripping over this, but TensorFlow 2.11.0 is the usual dependency in the `2022a` generation of easyconfigs, any reason you're not using that?

Reply:
I revised it for TF 2.11